packages/peregrine/lib/talons/MagentoRoute/useMagentoRoute.js

```diff
@@ -48,7 +48,10 @@ export const useMagentoRoute = (props = {}) => {
         routeData = { hasError: true, routeError };
     } else if (redirect) {
         // REDIRECT
-        routeData = { isRedirect: true, relativeUrl: relative_url };
+        routeData = {
+            isRedirect: true,
+            relativeUrl: relative_url.startsWith('/') ? "/" + relative_url : relative_url
```

Comment:
@Rykh Sorry! It would need to be
`!relative_url.startsWith('/') ? '/' + relative_url : relative_url`
Currently it would add the '/' twice

Reply:
Done